scripts/migrate_to_rapidpro/bulk_archive_chats.py

```diff
@@ -0,0 +1,144 @@
+import csv
+import json
+import time
+from datetime import datetime
+
+import pytz
+import requests
+from six.moves import urllib_parse
+from temba_client.v2 import TembaClient
+
+RAPIDPRO_URL = "https://rapidpro.qa.momconnect.co.za/"
+RAPIDPRO_TOKEN = ""
+
+TURN_URL = "https://whatsapp.turn.io"
+TURN_TOKEN = ""
+
+filename = "turn_open_chats.csv"
+today = datetime.utcnow().replace(tzinfo=pytz.utc)
+
+client = TembaClient(RAPIDPRO_URL, RAPIDPRO_TOKEN)
+
+
+def get_whatsapp_messages(wa_id):
+    headers = {
+        "Authorization": "Bearer {}".format(TURN_TOKEN),
+        "content-type": "application/json",
```

Comment:
I'm not sure if `requests` converts this, but maybe it should be title case for consistency?